cylc/flow/network/schema.py

```diff
@@ -1446,13 +1446,12 @@ class WorkflowStopMode(Enum):
 
     # NOTE: using a different enum because:
     # * Graphene requires special enums.
-    # * We only want to offer a subset of stop modes.
+    # * We only want to offer a subset of stop modes (REQUEST_* only).
 
-    # Note: contains only the REQUEST_* values from StopMode
-    Clean = StopMode.REQUEST_CLEAN.value
-    Kill = StopMode.REQUEST_KILL.value
-    Now = StopMode.REQUEST_NOW.value
-    NowNow = StopMode.REQUEST_NOW_NOW.value
+    Clean = StopMode.REQUEST_CLEAN
+    Kill = StopMode.REQUEST_KILL
+    Now = StopMode.REQUEST_NOW
+    NowNow = StopMode.REQUEST_NOW_NOW
```

Comment:
This reverts changes introduced by #4331, which set the values of the `WorkflowStopMode graphene.Enum` members to strings (the strings being the values of the `StopMode enum.Enum` members). But as far as I can tell, everything works if the values of `WorkflowStopMode` members are just the `StopMode` members, not their values.

Ugh, this is confusing